### PR TITLE
Replace pnpm with yarn in Nginx Proxy Manager LXC installation

### PR DESCRIPTION
**Pull request by mzonski**
## tl;dr
Because of faulty `pnpm` execution during an installment of *Ngnix Proxy Manager LXC* I have replaced it with `yarn` [as pointed here](https://github.com/NginxProxyManager/nginx-proxy-manager/blob/79cd0c5294f1e3ad4cc9ed5187b7759808b91162/frontend/yarn.lock#L2)

## Motivation
During the installation of the *Nginx Proxy Manager LXC* I stumbled upon an issue about a segmentation fault. After careful investigation, it happened that `pnpm` was throwing this error when trying to update packages. I have analyzed the Ngnix Proxy Manager frontend package and it is configured to use `yarn` by default, so I have adjusted the scripts responsible for the installation of the frontend. Moreover, I have applied clean install instead of install and removed package upgrade as it is generally not safe to bump packages automatically and we should use the same packages as the provider delivers in yarn.lock.

P.S. The issue with Seg fault was happening always until the writing of this PR to provide screenshots, nevertheless, the installation process will be even more perfect when using yarn instead of pnpm on the container.

### Fixes
- Installation of *Nginx Proxy Manager LXC*

### Type of change
- [X] Bug fix (non-breaking change which fixes an issue)
- [X] I have performed a self-review of my code, adhering to established codebase patterns and conventions.

### CODE REVIEW

**Comment by tteck** [comment]
I changed from `yarn` to `pnpm` here https://github.com/tteck/Proxmox/commit/edf3803362cb7ad152fb386997def01edf5ef57e (don't remember the issue). I have yet to have any segmentation fault. If anything, I need to increase the vCPUs to 2. If I can't replicate the issue, I can't justify changing back to `yarn`. 

```
    _   __      _               ____                           __  ___                                 
   / | / /___ _(_)___  _  __   / __ \_________ __  ____  __   /  |/  /___ _____  ____ _____ ____  _____
  /  |/ / __  / / __ \| |/_/  / /_/ / ___/ __ \| |/_/ / / /  / /|_/ / __  / __ \/ __  / __  / _ \/ ___/
 / /|  / /_/ / / / / />  <   / ____/ /  / /_/ />  </ /_/ /  / /  / / /_/ / / / / /_/ / /_/ /  __/ /    
/_/ |_/\__, /_/_/ /_/_/|_|  /_/   /_/   \____/_/|_|\__, /  /_/  /_/\__,_/_/ /_/\__,_/\__, /\___/_/     
      /____/                                      /____/                            /____/             
 
Using Advanced Settings
Using Distribution: debian
Using debian Version: 12
Using Container Type: 1
Using Root Password: Automatic Login
Container ID: 111
Using Hostname: nginxproxymanager
Using Disk Size: 4
Allocated Cores: 1
Allocated RAM: 1024
Using Bridge: vmbr0
Using IP Address: dhcp
Using Gateway IP Address: Default
Using APT-Cacher IP Address: Default
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using MAC Address: AE:1A:60:07:B7:59
Using Vlan: Default
Enable Root SSH Access: no
Enable Verbose Mode: no
Creating a Nginx Proxy Manager LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using thin-pool for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 111 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.250 
 ✓ IPv4 Internet Connected
 ✗ IPv6 Internet Not Connected
 ✓ DNS Resolved github.com to 140.82.114.3
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Installed Python Dependencies
 ✓ Installed Openresty
 ✓ Installed Node.js
 ✓ Installed pnpm
Would you like to install an older version (v2.10.4)? <y/N> 
 ✓ Downloaded Nginx Proxy Manager v2.11.2
 ✓ Set up Enviroment
 ✓ Built Frontend
 ✓ Initialized Backend
 ✓ Created Service
 ✓ Customized Container
 ✓ Started Services
 ✓ Cleaned
 ✓ Completed Successfully!

Nginx Proxy Manager should be reachable by going to the following URL.
         http://192.168.86.250:81
```

**Comment by tteck** [comment]
Can you share the exact error message?
